From 464942dd2f0bdaed3275ba2f216528227345c77b Mon Sep 17 00:00:00 2001
From: gbhakteX <gurudattax.bhakte@intel.com>
Date: Mon, 30 Jul 2012 12:26:58 +0530
Subject: [PORT FROM MAIN]Files with AMR_NB decoder and 3ga container could not be recognized.

BZ: 18401

Adding support for audio files with ".3ga" extension. Modified code in -
StagefrightMediaScanner.cpp - ".3ga" is added as a valid file extension.

Change-Id: Ie974beb106370e904b0a725879eccf89df7ffac0
Orig-Change-Id: Iaac6a36077fc6b18860b74fca245eff19900f605
Category: aosp improvement
Domain: AUDIO_MEDIA-Framework
Origin: internal
Upstream-Candidate: yes
Signed-off-by: gbhakteX <gurudattax.bhakte@intel.com>
---
 media/libstagefright/StagefrightMediaScanner.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/media/libstagefright/StagefrightMediaScanner.cpp b/media/libstagefright/StagefrightMediaScanner.cpp
index cd64ef8..5baad34 100644
--- a/media/libstagefright/StagefrightMediaScanner.cpp
+++ b/media/libstagefright/StagefrightMediaScanner.cpp
@@ -42,7 +42,7 @@ static bool FileHasAcceptableExtension(const char *extension) {
         ".mov", ".ra", ".rm", ".rmvb", ".ac3", ".ape", ".dts",
         ".mp1", ".mp2", ".f4v", "hlv", "nrg", "m2v", ".swf",
         ".avi", ".mpg", ".mpeg", ".awb", ".vc1", ".vob", ".divx",
-        ".mpga", ".mov", ".qcp", ".ec3", ".ts"
+        ".mpga", ".mov", ".qcp", ".ec3", ".ts", ".3ga"
     };
     static const size_t kNumValidExtensions =
         sizeof(kValidExtensions) / sizeof(kValidExtensions[0]);
-- 
1.7.4.1

